virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc

```diff
@@ -0,0 +1,25 @@
+[id="virt-configuring-vgpu-passthrough"]
+= Configuring vGPU passthrough 
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-vgpu-passthrough
+
+toc::[]
+
+Your cluster and virtual machines can access PCI host devices or mediated hardware devices through a virtual GPU (vGPU) passthrough configuration. Configuring a vGPU passthrough allows you to:
```

Comment:
Instead:
`Your virtual machines can access a virtual GPU (vGPU) hardware. Assigning a vGPU to your virtual machine allows you to:`

Reply:
Great, thanks.